src/wilson/dom.cljs

```diff
@@ -78,6 +79,38 @@
   ([ks rows]
    (table ks rows {})))
 
+
+(defn sorted-table
+  ([ks rows opts]
+   (let [state (r/atom {:sort-key (first ks)
+                        :sort-order :asc})
+         state-deref @state
+         sort-fns (or (:sort-fns opts)
```

Comment:
`sort-fns` is what gets passed to `sort-rows`: https://github.com/smogg/wilson/blob/sorted-table-component/src/wilson/dom.cljs#L42

`